targets/targets.json

```diff
@@ -5244,7 +5246,8 @@
             "SPI",
             "SPISLAVE",
             "TSC",
-            "MPU"
+            "MPU",
+	    "USTICKER"
```

Comment:
this should fix the build issue, one style change request: can you align usticker to the rest of the code (one space right)

Reply:
Hi Martin, this is a tab in the line before, I have changed it and it's aligned now.